.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

Wouldn't hurt to check on github actions too ✌
```suggestion
  open-pull-requests-limit: 10
- package-ecosystem: github-actions
  directory: "/"
  schedule:
    interval: daily
  open-pull-requests-limit: 10
```

---

What does this do?

---

It's checking on action versions used in workflows, just like you bumped actions in #4845.

Docs: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot